_component_design/badge.md

```diff
@@ -18,32 +18,106 @@ pagelink: https://rackerlabs.github.io/helix-ui/components/badges/
 
 <section class="static-section" markdown="1">
 
-## Introduction
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Use a badge when you want to signal to the user that a new notification, alert, message, chat, comment, or other type of change has taken place. A badge can appear in a navigation menu, tab, or the body of a control panel page.
+
+Use badges sparingly as overuse can lead to *notification fatigue* where the user fails to react to the badge.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Best Practices
+
+Consider the following best practices when you construct badges:
+
+- When the user clicks the badge, the user is taken to the section to which the badge is applied.
```

Comment:
the user moves to the section
(active voice rather than passive)

Reply:
Done.